### ISSUE TITLE
Naming conflict with `git-extras`

### ISSUE THREAD

**Issue by schlessera** [Originate/git-town#738]
`git-town` fails to install through `brew install git-town` because of a naming conflict with `git-extras`:

```
$ brew install git-town
Error: Cannot install git-town because conflicting formulae are installed.

  git-extras: because git-town also ships a git-sync binary

Please `brew unlink git-extras` before continuing.

Unlinking removes a formula's symlinks from /usr/local. You can
link the formula again after the install finishes. You can --force this
install, but the build may fail or cause obscure side-effects in the
resulting software.
```

**Comment by schlessera** [comment]
Here's what a dry run tells me:

```
$ brew link --overwrite --dry-run git-town
Would remove:
/usr/local/bin/git-sync -> /usr/local/Cellar/git-extras/4.1.0/bin/git-sync
/usr/local/share/man/man1/git-sync.1 -> /usr/local/Cellar/git-extras/4.1.0/share/man/man1/git-sync.1
```

**Comment by charlierudolph** [comment]
Yeah we've had issues with naming conflicting with them before. @kevgo thoughts on moving forward with #515 in order to resolve this? I also think it would beneficial if git extras did something similar.

**Comment by kevgo** [comment]
Yes, let's implement #515. Git-Extras will keep breaking us as they add more Git Town-like commands.

**Comment by charlierudolph** [comment]
Fixed in 2.0.0 which namespaces all commands with `town-`

**Comment by kevgo** [comment]
Thanks for notifying us about this @schlessera!
